### DISCUSSION TITLE
Is it possible to copy/paste style of a component

### DISCUSSION THREAD

**Discussion by javadsamiee** [GrapesJS/grapesjs#4836] [Q&A]
I want to add a feature to copy styles of a block/component and apply to another one. Something like "Copy layer style" on Photoshop.

<img width="426" alt="image" src="https://user-images.githubusercontent.com/65819265/211564461-aee8ac71-8116-4bf8-b98b-f5aa7101322b.png">

**Comment by ronaldohoch** [comment]
Hello!
You can get the applied style using this:
`editor.getSelected().attributes.style`

And apply to the new element using something like this:
`editor.CssComposer.setRule('.element-class', { 'css-rule': 'css-value' })`
